scripts/build.js

```diff
@@ -30,6 +31,23 @@ const runWebpack = config => {
   });
 };
 
+const compileAll = async () => {
+  webpackConfigs.forEach(async config => {
+    await runWebpack(config);
+  });
+};
+
+const cleanDistFolders = async () => {
+  builds.forEach(async ({ paths }) => {
+    console.log(`Clearing ${paths.dist}`);
+    await rimraf(paths.dist, err => {
```

Comment:
This `rimraf` call isn't returning a promise, is it? Looks like it's just dealing in regular callbacks here.

Reply:
Good catch. Will convert it using bluebirds `Promisify`?

Reply:
Yep. We already depend on es6-promisify, you import it like this:

```js
const { promisify } = require('es6-promisify');
````